analyzer/codechecker_analyzer/analyzers/clangtidy/analyzer.py

```diff
@@ -320,15 +354,21 @@ def get_checker_list(self, config) -> Tuple[List[str], List[str]]:
                 elif warning_type == CheckerType.analyzer:
                     if state == CheckerState.enabled:
                         compiler_warnings.add('-W' + warning_name)
-                    elif state == CheckerState.disabled:
-                        checkers.append('-' + checker_name)
+                        enabled_checkers.add(checker_name)
```

Comment:
I suppose the comments above are outdated.

Reply:
Which part do you think is outdated?